### PR TITLE
chore(README): add deprecation notice

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

**Review by Abhinandan-Purkait — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,22 @@
...
+## ⚠️ Deprecation Notice: Helm Chart Registry Migration
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
+
+**Old Registry URL:** https://openebs.github.io/charts  
+**New Registry URL:** https://openebs.github.io/openebs  
+
+To ensure seamless access to OpenEBS Helm charts, please update your configurations to use the new registry URL.
+
+Additionally, the older registry will be relocated to the 'github.com/openebs-archive' GitHub organization shortly.
```
Comment (avishnu) [inline README.md:20]: replace shortly with SIX months

File: README.md [hunk 2]
```diff
@@ -8,6 +8,22 @@
...
 
+## ⚠️ Deprecation Notice: Helm Chart Registry Migration
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
+
+**Old Registry URL:** https://openebs.github.io/charts  
+**New Registry URL:** https://openebs.github.io/openebs  
+
+To ensure seamless access to OpenEBS Helm charts, please update your configurations to use the new registry URL.
+
```
Comment (balaharish7) [inline README.md:19]: Avoid using the term "please" throughout the documentation.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,22 @@
...
+Additionally, the older registry will be relocated to the 'github.com/openebs-archive' GitHub organization shortly.
+
+For more information and guidance on this migration, please refer to the [OpenEBS documentation](https://openebs.io/docs).
+
```
Comment (balaharish7) [inline README.md:23]: Rephrase this as: 
Refer to the [OpenEBS documentation](https://openebs.io/docs) for more information and guidance on this migration.

File: README.md [hunk 4]
```diff
@@ -8,6 +8,22 @@
...
+Thank you for your attention to this matter. If you have any questions or need assistance with the migration, please reach out to the OpenEBS community.
+
```
Comment (balaharish7) [inline README.md:25]: We can provide the OpenEBS Community link here.

File: README.md (hunk 1 above)
Comment (niladrih) [reply README.md]: Added the relocation deadline.

File: README.md (hunk 2 above)
Comment (niladrih) [reply README.md:19]: Done.

File: README.md (hunk 4 above)
Comment (niladrih) [reply README.md:27]: Added the link https://openebs.io/community.

**Review by avishnu — APPROVED** [review]
Lgtm
